csp-mixin/alerts/gcp-alerts.yml

```diff
@@ -14,3 +14,129 @@ groups:
+        for: 5m
+        keep_firing_for: 10m
+        labels:
+          severity: critical
+          service: 'Cloud SQL'
+          namespace: cloud-provider-gcp
+        annotations:
+          summary: 'Memory utilization is too high.'
+          description: 'Review high-memory queries or add more memory to the {{ $labels.database_id }} instance.'
+          dashboard_uid: 'cc710d49022fdd69bed0e992891863e9'
+
+      - alert: GcpCloudSQLDiskUsage
+        expr: |
+          100 * avg by (job,project_id,instance,database_id) (stackdriver_cloudsql_database_cloudsql_googleapis_com_database_disk_utilization{job=~".+",project_id=~".+",instance=~".+", database_id=~".+"}) > 85
+        for: 5m
+        keep_firing_for: 10m
+        labels:
+          severity: critical
+          service: 'Cloud SQL'
+          namespace: cloud-provider-gcp
+        annotations:
+          summary: 'Disk utilization is too high.'
+          description: 'Delete or archive unused data, or increase disk size to the {{ $labels.database_id }} database.'
+          dashboard_uid: 'cc710d49022fdd69bed0e992891863e9'
+
+      - alert: GcpCloudSQLActiveConnections
+        expr: |
+          avg by (job,project_id,instance, database_id) (stackdriver_cloudsql_database_cloudsql_googleapis_com_database_mysql_threads{thread_kind="THREADS_CONNECTED", job=~".+",project_id=~".+",instance=~".+", database_id=~".+"}) > 0.9 * avg by (job,project_id,instance, database_id) (stackdriver_cloudsql_database_cloudsql_googleapis_com_database_mysql_max_connections{job=~".+",project_id=~".+",instance=~".+", database_id=~".+"})
+        for: 5m
+        keep_firing_for: 10m
+        labels:
+          severity: critical
+          service: 'Cloud SQL'
+          namespace: cloud-provider-gcp
+        annotations:
+          summary: 'Too many active connections.'
+          description: 'Investigate connection pooling settings and connection management in your {{ $labels.database_id }} application database.'
+          dashboard_uid: 'cc710d49022fdd69bed0e992891863e9'
+
+      - alert: GcpCloudSQLAbortedConnections
+        expr: |
+          sum by(job, instance, project_id)(rate(stackdriver_cloudsql_database_cloudsql_googleapis_com_database_mysql_aborted_connects_count[5m])) > 5
+        for: 5m
+        keep_firing_for: 10m
+        labels:
+          severity: critical
+          service: 'Cloud SQL'
+          namespace: cloud-provider-gcp
+        annotations:
+          summary: 'More than 5 failed connections in 5 minutes.'
+          description: 'Verify credentials and network settings; check for firewall rules blocking connections for the {{ $labels.database_id }} database.'
+          dashboard_uid: 'cc710d49022fdd69bed0e992891863e9'
+
+      - alert: GcpCloudSQLLagSecondsBehindMaster
+        expr: |
+          avg by (job,project_id,instance, database_id) (stackdriver_cloudsql_database_cloudsql_googleapis_com_database_mysql_replication_seconds_behind_master) > 5
```

Comment:
Are you sure this metrics return values in seconds?

Reply:
Yes, it contains the word `seconds` in its name...
Also, this is a screenshot from the docs:

![image](https://github.com/user-attachments/assets/08c6581e-95a0-4f49-85cd-8f8f3f8a701c)